src/opaque.c

```diff
@@ -202,7 +202,8 @@ static int prf(const uint8_t *pwd, const uint16_t pwd_len,
   return 0;
 }
 
-static int blind(const uint8_t *pw, const uint16_t pwlen, uint8_t *r, uint8_t *alpha) {
+// See https://libsodium.gitbook.io/doc/advanced/point-arithmetic/ristretto.
+static int blind(const uint8_t *pw, const uint16_t pwlen, uint8_t r[crypto_core_ristretto255_SCALARBYTES], uint8_t alpha[crypto_core_ristretto255_BYTES]) {
```

Comment:
very good, explicit param sizes! even if not a public fn still very helpful for reviewers and contributors!